transformer/blacklist.py

```diff
@@ -1,25 +1,28 @@
 import os
 import logging
+from typing import Sequence
 
+Blacklist = Sequence[str]
 
-def on_blacklist(url):
-    """
-    Checks for matching URLs in an ignore file (blacklist)
-    from user's current directory.
-    """
+
+def from_file() -> Blacklist:
     blacklist_file = f"{os.getcwd()}/.urlignore"
     try:
         with open(blacklist_file) as file:
-            blacklist = [line.rstrip("\n") for line in file if len(line) > 1]
-
-        for blacklist_item in blacklist:
-            if blacklist_item in url:
-                return True
-
-        return False
-
+            return [line.rstrip("\n") for line in file if len(line) > 1]
```

Comment:
I think that on files created on Windows, `len(line) > 1` will always be true because of `\r\n`.

```py
>>> lines = ["\n", " \n", "\r\n", " a\n", " a\r\n"]
>>> [line.rstrip("\n") for line in lines if len(line) > 1]
[' ', '\r', ' a', ' a\r']
>>> [line.rstrip() for line in lines]
['', '', '', ' a', ' a']
>>> list(filter(len, (line.rstrip() for line in lines)))
[' a', ' a']
```
Something like this is probably more general:
```suggestion
            return list(filter(len, (line.rstrip() for line in file)))
```